META.coq.in

```diff
@@ -459,28 +459,16 @@ package "plugins" (
     archive(native)  = "int31_syntax_plugin.cmx"
   )
 
-  package "asciisyntax" (
+  package "stringnotations" (
```

Comment:
Maybe "string_notations" would be easier to read? Unless the name is forced by other constraints?